addons/account/models/account_move.py

```diff
@@ -1456,7 +1458,7 @@ def _fetch_duplicate_supplier_reference(self, only_posted=False):
             **place_holders
         })
         return {
-            self.env['account.move'].browse(res['move_id']): self.env['account.move'].browse(res['duplicate_ids'])
+            res['move_id']: self.env['account.move'].browse(res['duplicate_ids'])
```

Comment:
Actually, I can't just do that in stable.
That would break outside code relying on this to be a recordset instead of an id.